drivers/sensor/CMakeLists.txt

```diff
@@ -41,4 +41,7 @@ add_subdirectory_ifdef(CONFIG_TMP007		tmp007)
 add_subdirectory_ifdef(CONFIG_TMP112		tmp112)
 add_subdirectory_ifdef(CONFIG_VL53L0X		vl53l0x)
 
+# Mockup sensor drivers
+add_subdirectory_ifdef(CONFIG_MOCKUP_BATTERY_SENSOR		mockup/battery)
```

Comment:
Similar to what you did with Kconfig, move this to `drivers/sensor/mockup/CMakeLists.txt`